Add awesome-ruby-lsp reference to docs (#3280)
Shopify/ruby-lsp: jekyll/index.markdown

- [Standard Ruby (from v1.39.1)](https://github.com/standardrb/standard/wiki/IDE:-vscode#using-ruby-lsp)

> Other community driven add-ons can be found in [awesome-ruby-lsp](https://github.com/johansenja/awesome-ruby-lsp) or by
> searching on [rubygems](https://rubygems.org/search?query=name%3A+ruby-lsp) for the `ruby-lsp` prefix.

Reviewer: ```suggestion
Other community add-ons can be found

- by
searching [rubygems](https://rubygems.org/search?query=name%3A+ruby-lsp) for the `ruby-lsp` prefix.
- in the community drive list of [awesome-ruby-lsp](https://github.com/johansenja/awesome-ruby-lsp)
```
